src/app/lib/lnd-http/index.ts

```diff
@@ -360,6 +360,16 @@ export class LndHttpClient {
     );
   };
 
+  getUtxos = (params: T.GetUtxosParams = { max_confs: 100000000 }) => {
+    // max_confs must be set to a number higher than the chain's current
+    // block height in order to get all utxos for a wallet
```

Comment:
Interesting quirk, what if it's not sent? What's the default behavior?

Reply:
Very strange indeed, I thought the same as well. If you don't supply a value it will default to 0 and only return UTXOs that have 0 confs. I picked this huge number cause the block_height of BTC/LTC won't get this high for many years to come